tests/test_latentdistributiontest.py

```diff
@@ -146,19 +175,31 @@ def test_pooled(self):
         ldt.fit(A1, A2)
 
     def test_distances_and_kernels(self):
+        np.random.seed(123)
+        A1 = er_np(20, 0.3)
+        A2 = er_np(100, 0.3)
         # some valid combinations of test and metric
-        with pytest.warns(None) as record:
-            for test in self.tests.keys():
-                ldt = LatentDistributionTest(test, self.tests[test])
-            ldt = LatentDistributionTest("hsic", "rbf")
-        assert len(record) == 0
+        # # would love to do this, but currently FutureWarning breaks this
+        # with pytest.warns(None) as record:
+        #     for test in self.tests.keys():
+        #         ldt = LatentDistributionTest(test, self.tests[test])
+        #         ldt.fit(A1, A2)
+        #     ldt = LatentDistributionTest("hsic", "rbf")
+        #     ldt.fit(A1, A2)
+        # assert len(record) == 0
+        for test in self.tests.keys():
+            ldt = LatentDistributionTest(test, self.tests[test])
+            ldt.fit(A1, A2)
+        ldt = LatentDistributionTest("hsic", "rbf")
+        ldt.fit(A1, A2)
         # some invalid combinations of test and metric
         with pytest.warns(UserWarning):
             ldt = LatentDistributionTest("hsic", "euclidean")
         with pytest.warns(UserWarning):
             ldt = LatentDistributionTest("dcorr", "gaussian")
         with pytest.warns(UserWarning):
             ldt = LatentDistributionTest("dcorr", "rbf")
+            ldt.fit
```

Comment:
what is this for?

Reply:
i was just checking if you're paying attention (read: i have no idea why it's there; deleted)